frontend/vue/components/Syllabus/SyllabusForm.vue

```diff
@@ -90,36 +100,70 @@ export default defineComponent({
           location: 'create-syllabus'
         }
       },
-      modules: [{
-        id: 1
-      }],
-      modulesCounter: 1
+      sections: [] as Section[],
```

Comment:
Correct me if I'm wrong @korgan00 , these sections are not related with the **syllabus** entity, aren't they?

Reply:
In fact, is not used anywhere. Probably I forgot to remove it, good catch.